panacea/grpc_client.go

```diff
@@ -32,7 +31,7 @@ func makeInterfaceRegistry() sdk.InterfaceRegistry {
 
 func NewGrpcClient(conf *config.Config) (GrpcClientI, error) {
 	log.Infof("dialing to Panacea gRPC endpoint: %s", conf.Panacea.GRPCAddr)
-	conn, err := grpc.Dial(conf.Panacea.GRPCAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
If this setting is removed, an error is likely to occur during SSL communication.
Would you like to test with https://testnet-grpc.gopanacea.org?

Reply:
If there is no error, I think you can delete it.

Reply:
When you see an example given by cosmos-sdk ([link](https://github.com/cosmos/cosmos-sdk/blob/main/docs/run-node/interact-node.md#programmatically-via-go)), they said `The Cosmos SDK doesn't support any transport security mechanism.`. Please refer this example and check if there is no error with our testnet as @gyuguen said :)

Reply:
Thanks, I add `grpc.WithUnsecure()` to `Dial` Option. It is now work without error when I test with https://testnet-grpc.gopanacea.org . Please check the new commit.